kafka/changestream/neo4j_insta_users-filter/filter_test.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/codeuniversity/smag-mvp/kafka/changestream"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestFilter(t *testing.T) {
+	validPayloadJSON := []byte("{\"id\":1,\"user_name\":\"TestUser\"}")
+	invalidPayloadJSON := []byte("{\"id\":\"1\",\"user_name\":\"TestUser\"}")
+
+	t.Run("create event with unmarshable json", func(t *testing.T) {
```

Comment:
FYI: 
When you rephrase this to describe the context; something like 
```suggestion
	t.Run("given a create event with unmarshable json", func(t *testing.T) {
```
The cases are a bit nicer to read :) 

- I find ` unmarshable`  a bit misleading - maybe just call it valid/invalid json 😅